ivy/functional/backends/paddle/elementwise.py

```diff
@@ -444,48 +309,31 @@ def multiply(
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
     x1, x2, ret_dtype = _elementwise_helper(x1, x2)
-    if x1.dtype in [paddle.int8, paddle.int16, paddle.uint8, paddle.float16]:
```

Comment:
do `int8`, `int16`, `uint8` and `float16` work with this function without any manual type casting?

Reply:
yes they don't seem work, I guess I can't depend on the documentations or the source code since some type checking is no explicit.